src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -50,6 +52,8 @@ export default class VivViewerLayer extends CompositeLayer {
         loaderSelection
       });
       if (tile) {
+        tile.data = NO_WEBGL2 ? to32BitFloat(tile.data) : tile.data;
+        tile.byteLength = tile.data[0].BYTES_PER_ELEMENT * tileSize ** 2;
```

Comment:
Can you comment on mutation happening these two lines. Why a hardcoded 2 and why is the `byteLength` mutated.

Reply:
This was for when I was using `maxCacheByteSize` to handle memory management but we don't we use that anymore.